frontend/src/components/Shared/Map/CrossingMap.js

```diff
@@ -232,7 +181,25 @@ class CrossingMap extends React.Component {
 
   onMapClick = e => {
     const { map } = this.state;
-    const features = map.queryRenderedFeatures(e.point);
+    const { showOpen, showClosed, showCaution, showLongterm } = this.props;
+
+    const width = 10;
+    const height = 10;
+    let layersToQuery = [];
+    if (showOpen) layersToQuery.push('openCrossings');
+    if (showClosed) layersToQuery.push('closedCrossings');
+    if (showCaution) layersToQuery.push('cautionCrossings');
+    if (showLongterm) layersToQuery.push('longtermCrossings');
+
+    const features = map.queryRenderedFeatures(
```

Comment:
Did `queryRenderedFeatures` become more reliable?

I thought we were considering using something like http://turfjs.org/docs/#pointsWithinPolygon instead.

```js
pointsWithinPolygon(
  this.state.visibleCrossings.map(getCrossingAsPoint),
  this.state.mapBounds
)
```